### PR TITLE
Adds SwiftLint to Form

### PR DESCRIPTION
**Pull request by iZNunev**
This PR adds SwiftLint to Form and fixes warnings

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  - shorthand_operator
+
+included:
+  - Form
+  - FormTests
+  - Examples
+
+identifier_name:
+  excluded: [id, i, vc, at, to, l, r, j,
+shouldRecognizeSimultaneouslyWithOtherGestureRecognizer,
```
Comment (nataliq) [inline .swiftlint.yml]: Do we think these names are ok? If yes, maybe we need to increase the max identifier length, if no.. we better rename them than excluding them. What do you think?

Comment (iZNunev) [reply .swiftlint.yml]: Yeah, I spent some time thinking of other names but couldn't really reduce it below 40 characters, think increasing the max length would work, how much should I put?

Comment (nataliq) [reply .swiftlint.yml]: Whatever fits the current cases I guess :} And then we're a bit more conscious.

**Comment by iZNunev** [comment]
@nataliq is this good to go?

**Pull request merged**